### ISSUE TITLE
calling .close twice

### ISSUE THREAD

**Issue by yoshuawuyts** [datproject/dat-node#85]
You see:

```
bundle.js:784 TypeError: Cannot read property 'port' of null
    at Swarm.leave (/Users/anon/src/datproject/dat-desktop/node_modules/discovery-swarm/index.js:144:47)
    at Dat.leave.Dat.leaveNetwork (/Users/anon/src/datproject/dat-desktop/node_modules/dat-node/dat.js:75:16)
    at Dat.close (/Users/anon/src/datproject/dat-desktop/node_modules/dat-node/dat.js:110:8)
    at Manager.remove (/Users/anon/src/datproject/dat-desktop/bundle.js:591:9)
    at /Users/anon/src/datproject/dat-desktop/bundle.js:881:17
    at Array.map (native)
    at Object.model.effect [as deleteConfirm] (/Users/anon/src/datproject/dat-desktop/bundle.js:879:10)
    at Function.<anonymous> (/Users/anon/src/datproject/dat-desktop/node_modules/barracks/index.js:254:39)
    at /Users/anon/src/datproject/dat-desktop/node_modules/nanotick/index.js:37:14
    at /Users/anon/src/datproject/dat-desktop/node_modules/nanotick/index.js:50:28
```

it'd be cool if it either:

1) didnt care
2) said you cant close me twice

**Comment by joehand** [comment]
published in `1.3.2`. Throws an error if you try to close twice.

**Comment by joehand** [comment]
Actually we need to make this better. If you try to close it again before the first close finishes you'll still get this error.

Add a test for that!
